test/fast/Running 'nvm-exec' should display required node version

```diff
@@ -16,3 +16,4 @@ No NODE_VERSION provided; no .nvmrc file found";
 
 # Skip install, we want to test the error message
 diff <(echo "${EXPECTED}") <(echo "${OUTPUT}")
+rm .nvmrc
```

Comment:
aha, good catch. note tho that this doesn't actually restore what might already be present - we should back up and restore the existing `.nvmrc` in setup/teardown files

i'll take a crack at this part tho